packages/api-gateway/__tests__/src/drivers/http/routes/chats/index.test.js

```diff
@@ -0,0 +1,48 @@
+// Internal dependencies
+const { buildServer } = require('../../../../../../src/drivers/http/server');
+
+describe('GET /chats', () => {
+  const server = buildServer({ logger: false });
+
+  it('should return the list of chat rooms of the requesting user', async () => {
+    const response = await server.inject({
+      method: 'GET',
+      url: '/chats',
+      headers: {
+        authorization: 'Bearer 123',
```

Comment:
Ok, I will remove this for now because it is not associated with any use case and when we implement the bearer token validation I will add the library that you suggested.